balancer/rls/internal/keys/builder.go

```diff
@@ -120,6 +120,10 @@ type KeyMap struct {
 // RLSKey builds the RLS keys to be used for the given request, identified by
 // the request path and the request headers stored in metadata.
 func (bm BuilderMap) RLSKey(md metadata.MD, host, path string) KeyMap {
+	// The path passed in is of the form "/service/method". The keyBuilderMap is
```

Comment:
(How) do we know it _always_ begins with a leading slash?

Using `strings.Trim(service, "/")` instead of slice operations would solve that problem if it is technically possible.

Reply:
This function is only called from the picker and the value passed for `path` is `balancer.PickInfo.FullMethodName`, which is documented to be of the format `/service/method`. This is a valid assumption to make, right?

Also, I build the `keyBuilderMap` with keys of the form `/service/` or `/service/method` specifically for this purpose. I *could* build the map with keys of the form `service/` or `service/method`, do a `strings.TrimPrefix(path, "/")` at the top of this function. But I'm not sure if that buys us much.